setup-cxx/setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+# Safeguard in case this accidentally runs on Windows or macOS.
+if [ "$RUNNER_OS" != "Linux" ]; then
+	echo "On $RUNNER_OS, doing nothing."
+	exit
+fi
+
+sudo apt-get install --no-install-recommends g++-8 gcc-8
+
+DIR="$RUNNER_TEMP/gcc-override"
+mkdir "$DIR"
+ln -s /usr/bin/gcc-8 "$DIR/gcc"
```

Comment:
Isn't there a command for setting this up?

Reply:
It doesn't look like `/usr/bin/gcc` ordinarily participates in Debian's alternatives system.